### PR TITLE
Performance boosts 🚀 

### PR DESCRIPTION
**Pull request by maximusunc**
- Convert internal Strider to stay in reasoner-pydantic
- Make kp timeout an env var with a default of 10 seconds
- Fix bug where aux graphs aren't passed along

### CODE REVIEW

File: strider/fetcher.py [hunk 1]
```diff
@@ -1,14 +1,4 @@
-"""Fetcher 2.0.
-
- .oooooo..o     .             o8o        .o8
```
Comment (kennethmorton) [inline strider/fetcher.py:3]: RIP asci art!

File: strider/fetcher.py [hunk 2]
```diff
@@ -118,7 +129,7 @@ async def generate_from_kp(
             onehop_response = await kp.solve_onehop(
                 onehop_qgraph,
             )
-            await save_kp_onehop(kp.id, onehop_qgraph, onehop_response)
+            await save_kp_onehop(kp.id, onehop_qgraph, onehop_response.dict())
```
Comment (kennethmorton) [inline strider/fetcher.py:132]: why dict() here? should we defer that until later?

File: strider/server.py [hunk 3]
```diff
@@ -1,4 +1,12 @@
-"""Simple ReasonerStdAPI server."""
+"""
```
Comment (kennethmorton) [inline strider/server.py:1]: Oh good it lives on!

File: strider/server.py [hunk 4]
```diff
@@ -445,47 +457,53 @@ async def lookup(
...
+        async for result_kgraph, result, result_auxgraph in fetcher.lookup(None):
             # Message parsing also normalizes kgraph edge ids and updates result and aux graph edge ids
-            result_message = Message.parse_obj(
-                {
-                    "knowledge_graph": result_kgraph_dict,
-                    "results": [result],
-                    "auxiliary_graphs": result_auxgraph,
-                }
-            )
+            # result_message = Message.parse_obj(
```
Comment (kennethmorton) [inline strider/server.py:473]: We need this?

File: strider/server.py [hunk 5]
```diff
@@ -445,47 +457,53 @@ async def lookup(
...
             # Update the aux graphs
-            output_auxgraphs.update(result_message.auxiliary_graphs)
+            output_auxgraphs.update(result_auxgraph)
 
             # Update the results
             # hashmap lookup is very quick
-            sub_result = next(iter(result_message.results))
-            sub_result_hash = hash(sub_result)
-            existing_result = output_results.get(sub_result_hash, None)
+            # sub_result = next(iter(result))
```
Comment (kennethmorton) [inline strider/server.py:489]: kill off some commented out code here?

**Review by kennethmorton — APPROVED** [review]
This is a good one. 🫧🪣🧹🧽

File: strider/fetcher.py (hunk 1 above)
Comment (maximusunc) [reply strider/fetcher.py:3]: No! It was just moved to the server.py file!

File: strider/fetcher.py (hunk 2 above)
Comment (maximusunc) [reply strider/fetcher.py:132]: This is where we save the response to the cache. I'm not aware that a reasoner-pydantic object can be gzipped and then saved into redis.

File: strider/server.py (hunk 3 above)
Comment (maximusunc) [reply strider/server.py:1]: lol

File: strider/fetcher.py (hunk 2 above)
Comment (kennethmorton) [reply strider/fetcher.py:132]: 🫡

File: strider/server.py (hunk 4 above)
Comment (maximusunc) [reply strider/server.py]: Nope.

File: strider/server.py (hunk 5 above)
Comment (maximusunc) [reply strider/server.py]: Done.

**Pull request merged**